lib/fastclick.js

```diff
@@ -93,7 +93,7 @@
 		 *
 		 * @type number
 		 */
-		this.tapDelay = options.tapDelay || 200;
+		this.tapDelay = options.tapDelay !== undefined ? options.tapDelay : 200;
```

Comment:
We should probably use `Number.isInteger` here to ensure that we only accept integers